src/mixins/legal-api-mixin.ts

```diff
@@ -167,21 +168,57 @@ export default class LegalApiMixin extends Vue {
   }
 
   /**
-   * Fetches the org persons of the current business.
+   * Fetches the directors of the current business.
    * @returns a promise to return the data
    */
-  async fetchOrgPersons (orgPersonType: OrgPersonTypes): Promise<OrgPersonIF[]> {
+  async fetchDirectors (): Promise<OrgPersonIF[]> {
```

Comment:
New method, instead of overloading the previous method and then having to deal with a list of people that might be parties or might be directors (with different object properties). Now's it's 1 list with a common interface. The code below converts from one to the other.

(Ideally, the API would return the directors list in the common schema, but oh well.)